packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -177,13 +209,17 @@ export const PaymentUpdateForm = ({
           </AlertBar>
         )}
 
-        {paypal_payment_error && (
+        {!transactionInProgress && paypal_payment_error && (
           <AlertBar className="alert alertError">
             {errorAlertBarContent}
```

Comment:
Inline `getPaypalErrorAlertBarContent()` here so that it is not called until the other two conditions are true?

Reply:
That function doesn't create the React components unless there's a `paypal_payment_error`, but I could also avoid calling it entirely instead of every render, so I'll change it.